contracts/UserProxy.sol

```diff
@@ -145,14 +145,11 @@ contract UserProxy is Multicall {
     function toAMM(bytes calldata _payload) external payable {
         require(isAMMEnabled(), "UserProxy: AMM is disabled");
 
-        (bool callSucceed, ) = ammWrapperAddr().call{ value: msg.value }(_payload);
-        if (callSucceed == false) {
-            // Get the error message returned
+        (bool callSucceed, bytes memory result) = ammWrapperAddr().call{ value: msg.value }(_payload);
```

Comment:
Maybe we don't need to assign return data to `result`? Because we are still going to remove the prefixed byte (length of `result` data) in assembly (`revert(add(result, 0x20), returndatasize())`) and we are not using the `result` anyway.